src/julia.h

```diff
@@ -1682,6 +1682,7 @@ JL_DLLEXPORT jl_sym_t *jl_get_ARCH(void) JL_NOTSAFEPOINT;
 JL_DLLEXPORT jl_value_t *jl_get_libllvm(void) JL_NOTSAFEPOINT;
 extern JL_DLLIMPORT int jl_n_threadpools;
 extern JL_DLLIMPORT _Atomic(int) jl_n_threads;
+extern int jl_n_gcthreads;
```

Comment:
```suggestion
extern JL_DLLIMPORT int jl_n_gcthreads;
```

Do we really need to export this variable, or Do we really want to allow the user to modify the number of GC threads at runtime?
Perhaps this variable could be moved to `julia_internal.h`.